src/resources/views/base/inc/scripts.blade.php

```diff
@@ -1,6 +1,10 @@
 @if (config('backpack.base.scripts') && count(config('backpack.base.scripts')))
     @foreach (config('backpack.base.scripts') as $path)
-    <script type="text/javascript" src="{{ asset($path).'?v='.config('backpack.base.cachebusting_string') }}"></script>
```

Comment:
Note that with this, we're no longer using the cachebusting string. Should we remove it from the config file? If we remove this, do we have a _different_ way of cachebusing, when using Bassets? 🐶

Reply:
Moved question to issue https://github.com/Laravel-Backpack/bassets/issues/6